x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MlInitializationService.java

```diff
@@ -221,8 +221,9 @@ private void makeMlInternalIndicesHidden() {
                 logger.warn("One or more of the ML internal indices could not be made hidden.");
                 return;
             }
-            GetAliasesRequest getAliasesRequest = new GetAliasesRequest().indices(mlHiddenIndexPatterns)
-                .indicesOptions(IndicesOptions.LENIENT_EXPAND_OPEN_CLOSED_HIDDEN);
+            GetAliasesRequest getAliasesRequest = new GetAliasesRequest(MasterNodeRequest.INFINITE_MASTER_NODE_TIMEOUT).indices(
```

Comment:
Specified an infinite timeout here as this is an internal request - as per https://github.com/elastic/elasticsearch/issues/107984.